torch/csrc/jit/test_jit.cpp

```diff
@@ -296,7 +296,7 @@ void fromQualStringTests() {
   REQUIRE(Symbol::fromQualString("onnx::LSTM") == Symbol::onnx("LSTM"));
   REQUIRE(Symbol::fromQualString("attr::value") == Symbol::attr("value"));
   REQUIRE(Symbol::fromQualString("scope::") == Symbol::scope(""));
-  auto bad_inputs = {"scope", "foo::bar", "prim:Param", "::", ":", ""};
+  auto bad_inputs = {"scope", ":", ""};
```

Comment:
Since "foo::bar", "prim:Param", "::" are now valid inputs, can we have tests for them?